pysigner/mesosphere_signer.py

```diff
@@ -182,7 +190,7 @@ def medianize(prices: List[float]) -> int:
 
 
 class TellorSigner:
-    def __init__(self, cfg):
+    def __init__(self, cfg, private_key):
```

Comment:
`private_key=None` because I was getting an error for tests if I just want to run the mesosphere_signer once